a few msftidy things to clean up:
```
modules/exploits/linux/http/ueb9_api_storage.rb - [WARNING] Invalid CVE format: 'CVE-2017-12478'
modules/exploits/linux/http/ueb9_api_storage.rb - [ERROR] Incorrect disclosure date format
modules/exploits/linux/http/ueb9_api_storage.rb - [INFO] Please add a newline at the end of the file
modules/exploits/linux/misc/ueb9_bpserverd.rb - [WARNING] Invalid CVE format: 'CVE-2017-12477'
modules/exploits/linux/misc/ueb9_bpserverd.rb - [ERROR] Incorrect disclosure date format
modules/exploits/linux/misc/ueb9_bpserverd.rb:18 - [ERROR] Unicode detected: " issue to execute arbitrary commands with root privilege on the target system.\xE2\x84\xA2\n"
```
CVEs should have the first "CVE-" removed.
Check disclosure date vs other modules.

--- a/modules/exploits/linux/http/ueb9_api_storage.rb
+++ b/modules/exploits/linux/http/ueb9_api_storage.rb
@@ -24,7 +24,7 @@ class MetasploitModule < Msf::Exploit::Remote
       },
       'Author'         =>
         [
-          'Cale Smith'    # @0xC413
+          'Cale Smith',    # @0xC413
           'Benny Husted', # @BennyHusted
           'Jared Arave'   # @iotennui
         ],

--- a/modules/exploits/linux/http/ueb9_api_storage.rb
+++ b/modules/exploits/linux/http/ueb9_api_storage.rb
@@ -36,7 +36,7 @@ class MetasploitModule < Msf::Exploit::Remote
         [
           ['URL', 'https://support.unitrends.com/UnitrendsBackup/s/article/ka640000000TO5PAAW/000005756'],
           ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2017-12478'],
-          ['CVE', 'CVE-2017-12478'],
+          ['CVE', '2017-12478'],
         ],
       'Targets'        =>
         [
@@ -47,7 +47,7 @@ class MetasploitModule < Msf::Exploit::Remote
      { 'PAYLOAD' => 'linux/x86/meterpreter/reverse_tcp',
        'SSL' => true
     },
-      'DisclosureDate'  => 'Aug 8th 2017',
+      'DisclosureDate'  => 'Aug 8 2017',
       'DefaultTarget'   => 0))
     register_options(
       [

--- a/modules/exploits/linux/misc/ueb9_bpserverd.rb
+++ b/modules/exploits/linux/misc/ueb9_bpserverd.rb
@@ -15,7 +15,7 @@ class MetasploitModule < Msf::Exploit::Remote
       'Description'    => %q{
        It was discovered that the Unitrends bpserverd proprietary protocol, as exposed via xinetd,
        has an issue in which its authentication can be bypassed.  A remote attacker could use this
-       issue to execute arbitrary commands with root privilege on the target system.™
+       issue to execute arbitrary commands with root privilege on the target system.
       },
       'Author'         =>
         [
@@ -31,7 +31,7 @@ class MetasploitModule < Msf::Exploit::Remote
         [
           ['URL', 'https://support.unitrends.com/UnitrendsBackup/s/article/ka640000000CcZeAAK/000005755'],
           ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2017-12477'],
-          ['CVE', 'CVE-2017-12477'],
+          ['CVE', '2017-12477'],
         ],
       'Targets'        =>
         [
@@ -42,7 +42,7 @@ class MetasploitModule < Msf::Exploit::Remote
      { 'PAYLOAD' => 'linux/x86/meterpreter/reverse_tcp',
        'SSL' => false
     },
-      'DisclosureDate'  => 'Aug 8th 2017',
+      'DisclosureDate'  => 'Aug 8 2017',
       'DefaultTarget'   => 0))
     register_options(
       [

